src/boardHelper.ts

```diff
@@ -84,7 +89,12 @@ const createRow = (type: RowTypes): BoardRow<SquareType> => {
 export const createNewBoard = (): Board => {
   const array = Array.from(Array(BOARD_WIDTH).keys());
   const board = array.reduce((map: Board, index) => {
-    map[index] = index % 2 === 0 ? createRow(RowTypes.Mixed) : createRow(RowTypes.Walls);
+    map[index] =
+      index % 2 === 0
+        ? index === 0
+          ? createRow(RowTypes.Mixed, true)
```

Comment:
No need for this confusing ternary operator chaining. Just do `createRow(RowTypes.Mixed, index === 0)`